codegen/scala-gen/src/main/scala/com/akkaserverless/codegen/scalasdk/impl/ValueEntityTestKitGenerator.scala

```diff
@@ -164,4 +166,70 @@ object ValueEntityTestKitGenerator {
           |""".stripMargin)
   }
 
+  def integrationTest(
+      main: FullyQualifiedName,
+      valueEntity: ModelBuilder.ValueEntity,
+      service: ModelBuilder.EntityService): File = {
+
+    val client = FullyQualifiedName(service.fqn.name + "Client", service.fqn.parent)
+
+    implicit val imports: Imports =
+      generateImports(
+        Seq(main, valueEntity.state.fqn, client) ++
+        service.commands.map(_.inputType) ++
+        service.commands.map(_.outputType),
+        service.fqn.parent.scalaPackage,
+        otherImports = Seq(
+          "com.akkaserverless.scalasdk.valueentity.ValueEntity",
+          "com.akkaserverless.scalasdk.testkit.ValueEntityResult",
+          "com.akkaserverless.scalasdk.testkit.AkkaServerlessTestkit",
+          "org.scalatest.matchers.should.Matchers",
+          "org.scalatest.wordspec.AnyWordSpec",
+          "org.scalatest.BeforeAndAfterAll",
+          "org.scalatest.concurrent.ScalaFutures",
+          "org.scalatest.time.Span",
+          "org.scalatest.time.Seconds",
+          "org.scalatest.time.Millis"),
+        packageImports = Seq(valueEntity.fqn.parent.scalaPackage),
+        semi = false)
+
+    val entityClassName = service.fqn.name
+
+    File(
+      service.fqn.fileBasename + "IntegrationSpec.scala",
+      s"""|package ${service.fqn.parent.scalaPackage}
+          |
+          |$imports
+          |
```

Comment:
unmanagement comment? 
```suggestion
          |$unmanagedComment
          |
```